app/models/geometric-property.js

```diff
@@ -62,6 +108,16 @@ export default class extends Model {
   @attr({ defaultValue: () => EmptyFeatureCollection })
   proposedGeometry;
 
+  @computed('canonical', 'proposedGeometry', 'annotations', 'data')
+  get isReadyToProceed() {
+    return isMeaningfulChange(this);
```

Comment:
why do you have to pass this into the function?

Reply:
`this` refers to the model context in that function... I'm not sure, I was hoping to extract that bit out for better unit testing, but it probably doesn't add anything when we could just unit test the whole model